frontend/app/src/components/home/RecommendedGroup.svelte

```diff
@@ -27,6 +27,7 @@
     }
 
     function joinGroup(group: GroupChatSummary) {
+        if (client.isReadOnly()) return;
```

Comment:
Again - don't like that it is the _handler_ that is restricted. I think the button should not be there instead.